client/src/commands/wrapLinesWithComment.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright © 2022-2025, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
```

Comment:
Sorry I didn't make it clear. For new file it should be only this year.
```suggestion
// Copyright © 2025, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
```